src/index.js

```diff
@@ -30,22 +17,73 @@ function renderHtml() {
     .sort((a, b) => a.index - b.index)
     .forEach((task) => {
       listParent.innerHTML += `
-      <li>
+      <li draggable=true>
         <div class="content">
           <input class="check" type="checkbox" ${
-            task.completed ? 'checked' : ''
-          }/>
+  task.completed ? 'checked' : ''
+}/>
           <input class="input" type="text" value='${
-            task.description
-          }' readonly />
+  task.description
+}' readonly />
         </div>
         <div class="actions">
           <span class="material-icons drag">more_vert</span>
-          <span class="material-icons hide">delete_outline</span>
+          <span class="material-icons hide" id="delete">delete_outline</span>
         </div>
       </li>
       `;
     });
 }
 
 renderHtml();
+
+// adding and removing
```

Comment:
- [ ] Congratulations on implementing the add and remove, but as per the requirement, kindly do it on a different file and import it
![image](https://user-images.githubusercontent.com/73140461/149193020-bfa1a12b-31c6-43fa-95aa-f0414ebacd7f.png)

- [ ] Also kindly implement the function to edit task descriptions
![image](https://user-images.githubusercontent.com/73140461/149193208-1138e25e-80af-4dc9-98a5-df6d448427f0.png)

Reply:
This issue is well resolved! ✔️